.pylint.rc

```diff
@@ -0,0 +1,58 @@
+[MASTER]
+jobs=4
+persistent=yes
+suggestion-mode=yes
+unsafe-load-any-extension=no
+
+# Blacklist files or directories (basenames, not paths)
+ignore=
+
+# blacklist files or directories by regex  (basenames, not paths)
+ignore-patterns=
+
+[EXCEPTIONS]
+
+overgeneral-exceptions=Exception,RaidenUnrecoverableError
+
+[BASIC]
+
+bad-names=foo,bar,baz,toto,tutu,tata
+good-names=i,j,k,_
+
+[LOGGING]
+
+logging-modules=logging,structlog
+
+[MESSAGES CONTROL]
+
+disable=all
+enable=no-value-for-parameter,too-many-format-args,no-member
+
+[REPORTS]
+
+reports=no
+score=no
```

Comment:
this just clears the output, no need for the tables and a score for the automated build (also I find it quite annoying when running locally)